boltdesignsystem/bolt
# DS-471 Teaser Updates

## Jira

https://pegadigitalit.atlassian.net/browse/DS-471

## Summary

Teaser design and features updates were done.

## Details

- Schema was updated with the `variant` prop with a `card` and `transparent` options.
- The `type` prop was deprecated.
- Schema was updated with instructions on the `time` prop.
- New demo page with a video modal opening was added.
- Docs for the Teaser-Time page were updated.
- Demo pages using an image as a signifier to open a modal were rewritten to use Video Thumbnail as a signifier

## How to test

Pull the branch. Check if the schema is updated correctly. Go to the new demo page with the video modal and check the docs.

## Release notes

- New prop `variant` with `enum: 'card', 'transparent'` was added.
- Prop `type` was deprecated.
- Video Thumbnail should be used when creating a teaser opening video modal
